### ISSUE TITLE
Tax Calculations don't seem to be correct, with 19% tax rate

### ISSUE THREAD

**Issue by wm-simon** [duncanmcclean/simple-commerce#340]
![image](https://user-images.githubusercontent.com/3351449/105399414-e295a200-5c23-11eb-9670-92dd6784e483.png)

The totals can't be right, but I have no idea why.

Code is form the starter kit:
```
           <div class="bg-gray-200 p-4">
                <h2 class="text-gray-600 text-sm mb-4">Your cart</h2>
            
                {{ sc:cart:items }}
                    <a class="flex items-center justify-between mb-2 text-sm" href="{{ product:url }}">
                        <span>{{ quantity }}x {{ product:title }}</span>
                        <span>{{ total }}</span>
                    </a>
                {{ /sc:cart:items }}
            
                {{ if {sc:cart:shippingTotal} !== 0 }}
                    <div class="flex items-center justify-between mb-2 text-sm">
                        <span>Shipping</span>
                        <span>{{ sc:cart:shippingTotal }}</span>
                    </div>
                {{ /if }}
            
                {{ if {sc:cart:taxTotal} !== 0 }}
                <div class="flex items-center justify-between mb-2 text-sm">
                    <span>Tax</span>
                    <span>{{ sc:cart:taxTotal }}</span>
                </div>
                {{ /if }}
            
                {{ if {sc:coupon:has} === true }}
                    <div class="flex items-center justify-between mb-2 text-sm">
                        <span>Coupon Discount</span>
                        <span>{{ sc:cart:couponTotal }}</span>
                    </div>
                {{ /if }}
            
                <div class="flex items-center justify-between mb-2 text-sm">
                    <span class="font-semibold">Grand Total</span>
                    <span class="font-semibold">{{ sc:cart:grandTotal }}</span>
                </div>
            </div>
```

**Comment by duncanmcclean** [comment]
That certainly looks wrong. Thanks for creating an issue!

**Comment by duncanmcclean** [comment]
Re-opening as my previous fix didn't actually seem to fix the issue.

**Comment by duncanmcclean** [comment]
There we go... finally fixed in [v2.1.32](https://github.com/doublethreedigital/simple-commerce/releases/tag/v2.1.32).
